src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java

```diff
@@ -5,6 +5,8 @@
 import static com.codeborne.selenide.Condition.hidden;
 import static com.codeborne.selenide.Condition.visible;
 import static com.codeborne.selenide.Selenide.$;
+import static org.junit.Assert.assertFalse;
```

Comment:
Please remove the static imports for `assertFalse` and `assertTrue` and call the functions like 

`Assert.assert...`

Until now we use this scheme at most of our test cases and I would like to have the same approach for similar things.